app/assets/javascripts/fae/_tables.js

```diff
@@ -267,6 +267,8 @@ Fae.tables = {
 
       Fae.tables.sizeFixedHeader();
     });
+
+    $toggle.trigger('click');
```

Comment:
I agree with James. It's adding cruft; instead, the `.active` should be added to `.collapsible` in the dummy app and the docs (and the Open All changed to Close All and `.close-all` added to `.collapsible-toggle`). Simulating the click will have unintended consequences later.